test.js

```diff
@@ -4,14 +4,41 @@ import inRange from 'in-range';
 import fn from './';
 
 test('promise', async t => {
+	t.plan(1);
 	const end = timeSpan();
 	await fn(50);
-	t.true(inRange(end(), 30, 70));
+	t.true(inRange(end(), 30, 70), 'should be delayed');
 });
 
 test('thunk', async t => {
+	t.plan(2);
 	const end = timeSpan();
 	const result = await Promise.resolve('foo').then(fn(50));
-	t.true(inRange(end(), 30, 70));
-	t.is(result, 'foo');
+	t.true(inRange(end(), 30, 70), 'should be delayed');
+	t.is(result, 'foo', 'should pass through the value');
```

Comment:
Not a fan of using the `should` convention. This seems clearer to me:

> passes through the value

Same with the others.